cmd/collect/cli/root.go

```diff
@@ -32,6 +32,10 @@ func RootCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			v := viper.GetViper()
 
+			if err := checkAndSetChroot(v.GetString("chroot")); err != nil {
```

Comment:
Should we check if we're running as root as well eg. https://github.com/replicatedhq/troubleshoot/blob/main/internal/util/util.go#L115

Reply:
oh good point. the go chroot syscall does a pretty good job of complaining if it can't chroot but friendlier messages are always better